src/browser-frontend/views/browser/window.jsx

```diff
@@ -10,23 +10,49 @@ CONDITIONS OF ANY KIND, either express or implied. See the License for the
 specific language governing permissions and limitations under the License.
 */
 
-import React, { PureComponent } from 'react';
+import React, { PureComponent, PropTypes } from 'react';
 import CSSModules from 'react-css-modules';
+import { connect } from 'react-redux';
+import Mousetrap from 'react-mousetrap-mixin';
+
+import PagesModelActions from '../../actions/pages-model-actions';
+
 
 import Styles from './window.css';
 import Chrome from './chrome';
 import Content from './content';
 
+@connect(() => ({
+}))
 @CSSModules(Styles, {
   allowMultiple: true,
 })
 export default class Window extends PureComponent {
   render() {
+    const keys = {};
+    const mousetrap = (<Mousetrap mousetrap={keys} />);
+
+    mousetrap.type.prototype.addMousetrap(['command+t', 'ctrl+t'], () => {
+      this.props.dispatch(PagesModelActions.addPage());
+    });
+    mousetrap.type.prototype.addMousetrap(['command+w', 'ctrl+w'], () => {
+      this.props.dispatch(PagesModelActions.removePage({}));
```

Comment:
If you insist on making `pageId` default to the currently selected paged id in the reducer, you can avoid having to pass in an empty object here with the following reducer signature:

`function removePage(state, { payload: { ... } = {} }) { ... `